tools/markdown-magic/package.json

```diff
@@ -35,7 +35,6 @@
 	},
 	"devDependencies": {
 		"@fluidframework/build-common": "^2.0.3",
-		"@fluidframework/eslint-config-fluid": "^3.2.0",
```

Comment:
Would be nice to have cjs linting too but this seems fine for now if it's not doing anything.

Reply:
I started working on it, but determined it was too much work for this PR. I'll follow up on it soon though.